.editorconfig

```diff
@@ -9,8 +9,12 @@ indent_style = space
 indent_size = 4
 insert_final_newline = true
 trim_trailing_whitespace = true
+eclint_block_comment_start = /*
```

Comment:
Ah, excuse me, another question: when googling, the only result for this is: https://github.com/conedevelopment/sprucecss/blob/main/.editorconfig 
Is there documentation for this? What is it doing exactly?

Reply:
All the thing you get from me are not opinionated/personal but base on real life incidents.
https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#ideas-for-domain-specific-properties
https://github.com/greut/eclint#features

Reply:
`block_comment_*` keys are necessary for `eclint` to recognize block comments.
```php
/**
 * Block comment
 */
```

Reply:
Without checking for EC violations you cannot exercise Total Control over every byte.

Reply:
😅 OK...didn't say it's opinionated. I haven't seen it before and literally the ONLY result when googling for `eclint_block_comment_start` was the link I posted. So, I just want to know what we're adding to my project here 🤷🏻‍♂️

And spontaneously I don't even really know what it is about from the two links you posted 🤔

Reply:
We could remove the `eclint_` prefix!

Reply:
What for?

Reply:
Googling for `block_comment_start` yields much more results :)

Reply:
Ah, now I get it. Then I'd say let's go without the `eclint_` prefix...and please `block_comment_start = /**` => two asterisk.

Reply:
Ready.